python/ml4ir/base/features/feature_config.py

```diff
@@ -335,6 +336,22 @@ def get_feature_by_node_name(self, name: str):
 
         raise KeyError("No feature with node name {} in FeatureConfig".format(name))
 
+    def get_old_ranking_score(self, name: str = FeatureConfigKey.OLD_RANKING_SCORE):
+        """
+        Getter method for the old ranking score feature in FeatureConfig object
+
+        Parameters
+        ----------
+        name : str
+            Name of the feature to fetch
+
+        Returns
+        -------
+        dict
+            Feature config dictionary for the name of the feature passed
+        """
+        return self.feature_config.get_feature(name)
```

Comment:
@mohazahran Why do you need this?

Reply:
I can pass here a default name to that feature "s"